code/src/components/Form.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react' 
+import Button from 'components/Button'
+import TextInputOne from 'components/TextInputOne'
+import RadioButton from 'components/RadioButton'
+import SelectList from './SelectList'
+import TextInputTwo from 'components/TextInputTwo'
+import Summary from 'components/Summary'
+
+const Form = () => {
+    const [currentPage, setCurrentPage] = useState(0)
+    const [radioValue, setRadioValue] = useState("")
+    const [dropDown, setDropDown] = useState("")
+    const [firstThought, setFirstThought] = useState("")
+    const [societysPortrayal, setSocietysPortrayal] = useState("")
+
+    const onPageChange = () => {
+        setCurrentPage (currentPage + 1)
+    }
+    const radioAnswers = ["Wiccan", "Pagan", "Eclectic", "Other"]
+    console.log(firstThought)
+    console.log(radioValue)
+    console.log(dropDown)
+    console.log(societysPortrayal)
```

Comment:
I would be a good idea to remove the console logs when you deploy the build folder.